registry/quilt_server/views.py

```diff
@@ -283,9 +293,11 @@ def wrapper(*args, **kwargs):
 
                     data = resp.json()
                     # TODO(dima): Generalize this.
-                    g.user = data.get('current_user', data.get('login'))
-                    assert g.user
-                    g.email = data['email']
+                    user = data.get('current_user', data.get('login'))
```

Comment:
why is there also user data in `login`?

Reply:
It's for Django and GitHub - looks like this part of OAuth2 is not standardized. (And GitHub Enterprise even lets you configure the field names.)

We could add a config option, too, though this was easier to start with.